dbux-graph-common/src/shared/GraphMode.js

```diff
@@ -0,0 +1,22 @@
+import Enum from '@dbux/common/src/util/Enum';
+import GraphType from './GraphType';
+
+const GraphModeObj = {
```

Comment:
There used to be 3 `GraphMode`s, which defines the Graphs should be enabled:
```
const EnabledGraphTypesByMode = {
  [GraphMode.Sync]: [GraphType.SyncGraph],
  [GraphMode.Async]: [GraphType.AsyncGraph],
  [GraphMode.AsyncFull]: [GraphType.AsyncGraph, GraphType.AsyncStack]
};
```
But later I decided to control `AsyncStack` in another state `GraphDocument.state.stackEnabled` rather than `GraphDocument.state.graphMode`, since we might want to see `AsyncStack` in both `Sync` and `Async` mode. So this becomes useless now. I'll remove this in next PR.